common/utils/src/main/scala/org/apache/spark/util/MavenUtils.scala

```diff
@@ -215,7 +215,9 @@ private[spark] object MavenUtils extends Logging {
         if (artifactInfo.getExt == "jar") {
           true
         } else {
+          // scalastyle:off
```

Comment:
let's drop all the changes in this file. We can disable the style check in the PR which introduces the new rule